content/articles/why-you-should-use-wakanda/index.md

```diff
@@ -0,0 +1,63 @@
+
+Wakanda is supported on Microsoft windows, cloud-ready on back-end and Linux. 
+
+### Features of Wakanda JavaScript Framework 
+There are very nice features which make Wakanda JavaScript stand out in this modern day. 
+
+#### The Wakanda database
+The Wakanda Database itself stands out for me, especially when it is paired perfectly with the Wakanda Server. It offers the same experience as MongoDb and CouchDb since it uses modern NoSQL with similar portability as SQLite. Wakanda JavaScript framework saves us stress and time since it is portable and ready to use. 
+
+#### The Wakanda server
+The Wakanda server runs and hosts the backend applications in the course of production and development. It is a multi-threaded HTTP server which incorporates server-side JavaScript APIs unmasked in REST. Everything needed is already written for the users in JavaScript. It also saves time since the user does not have to go into configuration tasks. The Wakanda server is also portable. It lowers the cost of both configuration and development by using pre-bundled components.
+
+##### The JavaScript application server
+The Wakanda JavaScript app server uses the V8 engine which operates on JavaScript runtime. It exhibits a rich synchronous API to hold filesystems, sockets remote databases and workers. It supports asynchronous software development style.
+
+##### The HTTP server
+The Wakanda HTTP server exposes the autogenerated Wakanda REST API. It is also responsible for serving static application files.
+
+##### The Data abstraction layer
+The data abstraction layer is essential for data security. It helps in data manipulation through use of a JavaScript API. It exposes an automated REST API to manipulate and access data.
+
+##### Authentication (directory)
+Authentication is a process which enables the app in verifying the users who are connected to a certain network resource. It helps in exposing both the REST API and server side JavaScript API. It is responsible for authentication of management sessions and users.
+
+#### The Wakanda studio
+The Wakanda studio helps to develop and test apps locally together with troubleshooting your environment. In this era where everyone is craving for a code editor, Wakanda has taken a different turn in coming up with an IDE. 
+
+Wakanda's main objective is to give a quick and easy way in project scaffolding.Scaffolding is programming method which aims at creating  database-backed software applications. Many software frameworks are lacking code editors where programmers may need to write down specifications which guides on how the application database is operated.The philosophy behind Wakanda studio which simplifies developers lives who have decided to work with Wakanda JavaScript stack, is called RAD (Rapid Application Development).
+
```

Comment:
The statement `Many software frameworks are lacking code editors where programmers may need to write down specifications which guides on how the application database is operated` is misguiding. 

Code editors have a little or no connection to frameworks. Please revise this statement to convey the intended meaning.